assets/javascript/downloads.js

```diff
@@ -421,8 +421,17 @@ function shouldDisplayDownload(download, displayedManufacturers, displayedMcufam
 
   if (downloadsSearch.searchTerm && downloadsSearch.searchTerm.length > 0 && shouldDisplay) {
     var regex = new RegExp(downloadsSearch.searchTerm, "gi");
-    var haystack = download.dataset.name + " " + download.dataset.id + " " + download.dataset.manufacturer + " " + download.dataset.mcufamily + " " + download.dataset.features;
-
+    var dataFields = [
+        download.dataset.name,
```

Comment:
Thanks for changing this to a list. Much easier to read and add things to if anything else ever pops up.